deploy/gateway/templates/podmonitor.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.metrics.podMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ include "gateway.fullname" . }}
+  labels:
+    {{- include "gateway.labels" . | nindent 4 }}
+    {{- with .Values.metrics.podMonitor.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
```

Comment:
Do we really need this? Would Prometheus scrape the namespace the `PodMonitor` is deployed in?

Reply:
Hmm, nope, this is just setting the default value explicitly. Prometheus would discover the pods in the same namespace as the `PodMonitor`